lib/almodovar.rb

```diff
@@ -16,14 +16,27 @@
 require 'almodovar/to_xml'
 
 module Almodovar
-
   class << self
+    DEFAULT_SEND_TIMEOUT = 30
+    DEFAULT_CONNECT_TIMEOUT = 30
+    DEFAULT_RECEIVE_TIMEOUT = 30
+
     def default_options
-      @default_options ||= {
-        :timeout => 120,
-        :connect_timeout => 30,
-        :user_agent => "Almodovar/#{Almodovar::VERSION}"
+      default = {
+        send_timeout: DEFAULT_SEND_TIMEOUT,
```

Comment:
would you keep the API like `timeout`, `connect_timeout` and `receive_timeout`?

IMO it doesn't matter if we keep the same interface because if we'd change the underneath library we could keep the same interface we proposed here. I'd like hear you about the above question.